src/Microsoft.ML.FastTree/TreeEnsemble/QuantileRegressionTree.cs

```diff
@@ -2,19 +2,25 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
+using System.Collections.Generic;
 using Microsoft.ML.Data;
 using Microsoft.ML.Internal.Utilities;
 using Microsoft.ML.Model;
 using Float = System.Single;
 
 namespace Microsoft.ML.Trainers.FastTree.Internal
 {
-    public class QuantileRegressionTree : RegressionTree
+    internal class QuantileRegressionTree : RegressionTree
     {
-        // Holds the labels of samped instances for this tree
+        /// <summary>
+        /// Holds the labels of samped instances for this tree. This value can be null when training, for example, random forest (FastForest).
```

Comment:
>samped [](start = 32, length = 6)

typo

Reply:
Ok.

---
In reply to: [252033149](https://github.com/dotnet/machinelearning/pull/2243#discussion_r252033149) [](ancestors = 252033149)